### ISSUE TITLE
Cards grid does not work in Boostrap5 theme

### ISSUE THREAD

**Issue by irinaz** [backdrop-contrib/paragraphs_aplenty#46]
Hi, 
We are using cards on various sites, and recently I have noticed that styles are not working on https://github.com/backdrop-contrib/bootstrap5_lite theme - see attached screenshots. 

It seems that the problem is coming from  combination of "row grid" classes and "g-col-md-4" vs "col-md-4".   What is the best way to approach this issue?  We have a patch for our local site, but I am not sure how to make it work in a more generic way. 

<img width="1441" alt="basis" src="https://github.com/user-attachments/assets/f5ab4e72-c0d1-4e40-82be-c1fbcd26bee3">

<img width="1442" alt="bootstrap5" src="https://github.com/user-attachments/assets/55e9fcea-2b87-4a83-812c-edfe73e67ba6">

**Comment by herbdool** [comment]
Thanks @irinaz I'm looking into this today.

**Comment by herbdool** [comment]
@irinaz I've got a PR which fixes things for me. I've tested a three-card grid on Basis, Bootstrap Lite, Bootstrap 5 Lite.

I saw a couple issues. The grid was messed up on Bootstrap. I fixed that by making the row class variable, instead of being `row grid`. By default it is `grid` and there's a fallback if someone sets a different grid prefix in settings.php (`paragraphs_aplenty_grid_class_prefix`).

The other one, which you've shown above, is that a card without an image doesn't expand fully. I've set the child `.content` div of `.paragraphs-item` to be full width which seems to fix it.
